src/commands/run.ts

```diff
@@ -56,8 +54,20 @@ export async function run(options: RunOptions) {
     log.error("Unexpected error thrown when running watchtower", error);
     exitCode = 1;
   } finally {
-    await api.stop();
-    await DBService.getInstance().close();
-    process.exit(exitCode);
+    stop(exitCode);
   }
 }
+
+/**
+ * Run actions required when stopping the watch-tower from run mode
+ * @param exitCode Exit code to return to the shell
+ */
+function stop(exitCode?: number) {
+  const log = getLogger("commands:stop");
+  log.info("Stopping Rest API server...");
+  ApiService.getInstance().stop();
```

Comment:
this is a promise. Shouldn't yuou await it? 

same as for the db service.
probably you should catch an error closing it.

Q: can the two services be closed in paralle? then you could do a promise. If so, probably you could use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled

Reply:
Great catch. Silly me. Will modify and push with the Promise.allSettled.